Source/ui_unix/PH_HidUnix.h

```diff
@@ -0,0 +1,74 @@
+#ifndef _PH_HIDUnix_H_
+#define _PH_HIDUnix_H_
+
+#include "PadHandler.h"
+#include <memory>
+
+class CPH_HidUnix : public CPadHandler
+{
+public:
+							CPH_HidUnix();
+	virtual					~CPH_HidUnix();
+	
+	void					Update(uint8*);
```

Comment:
Function should be marked as `override` since it overrides the function from `CPadHandler`.